completions/gcc

```diff
@@ -7,7 +7,7 @@ _comp_cmd_gcc()
 
     # Test that GCC is recent enough and if not fallback to
     # parsing of --completion option.
-    if ! $1 --completion=" " 2>/dev/null; then
+    if ! "$1" --completion=" " 2>/dev/null; then
```

Comment:
Cherry picked as 26d8b345eee07e15937219ca3908bfa15ac9caf9. Quoting is also necessary due to possible spaces in path to the command even with the standard `$IFS`.